src/main/java/book/shop/dto/CreateBookRequestDto.java

```diff
@@ -1,14 +1,26 @@
 package book.shop.dto;
 
+import book.shop.validation.Author;
+import book.shop.validation.Unique;
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
 import java.math.BigDecimal;
 import lombok.Data;
 
 @Data
 public class CreateBookRequestDto {
+    @NotBlank
     private String title;
+    @NotBlank
+    @Author
     private String author;
+    @NotNull
+    @Min(0)
     private BigDecimal price;
     private String description;
```

Comment:
you can validate description

Reply:
description is not required. Let's validate size instead. Unique is a bit weird here